### ISSUE TITLE
Feature Request: Create mod

### ISSUE THREAD

**Issue by poperigby** [limo-app/limo#55]
Sometimes I'll want to create a custom mod, like a collection of patches, but currently I have to zip the folder full of patches up and add them to LIMO. It would be useful to have a a "Create mod" button that simply adds an empty mod directory that you can copy files into.

**Comment by Heisenborgar** [comment]
Yeah, this is a must have feature. I think "Create mod" saves some time, rather than having to zipping the files themselves then changing zip compression to store for faster extract or having to manually modify the json mod list file.

**Comment by limo-app** [comment]
I like the idea and this should be easy enough to implement. Will most likely be part of 1.0.9.
